src/main/controllers/OtpLoginController.ts

```diff
@@ -0,0 +1,17 @@
+import { Request, Response } from 'express';
+
+export default class OtpLoginController {
+  public get(req: Request, res: Response): void {
+    res.render('otp-login');
+  }
+
+  public post(req: Request, res: Response): void {
+    if (req.body['otp-code'] !== undefined && req.body['otp-code'].length === 6) {
```

Comment:
I think `req.body['otp-code'] !== undefined` can be refactored into `req.body['otp-code']`

Reply:
Good spot :) I've updated that now